src/content/docs/style-guide/writer-workflow/github-intro.mdx

```diff
@@ -0,0 +1,154 @@
+      </td>
+    </tr>
+    <tr>
+      <td>
+        **Any TW: Writer needs PR review**
+      </td>
+      <td>
+        PRs from Tech Docs team members that need a light edit pass to make sure everything in GitHub is correct. This should be checked by other writers every few hours so PRs don’t get stale. 
+        
+        If you have a PR that’s been lingering here too long, ask for a reviewer in #doc_sprint_talk. 
+        
+        **Whoever takes it: assign yourself as Reviewer.**
+      </td>
+    </tr>
+    <tr>
+      <td>
+        **Any TW: needs peer edit**
+      </td>
+      <td>
+        Like our Needs Peer Edit column in Jira: A writer has requested a review of their PR. Review their PR in GitHub and leave comments. 
+        
+        **Whoever takes it: assign yourself as Reviewer.**
+      </td>
+    </tr>
+    <tr>
+      <td>
+        **Waiting on SME**
+      </td>
+      <td>
+        For PRs that are blocked by need for SME info or confirmation (for example, as Hero you are waiting on an answer from the person who sent in a Hero pull request).
+      </td>
+    </tr>
+    <tr>
+      <td>
+        **Waiting on TW to merge**
+      </td>
+      <td>
+        All reviews are complete. The TW who created the PR (or who is assigned the issue) needs to merge this work into develop.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        **Drafts**
+      </td>
+      <td>
+        A draft is a way to open a PR while indicating that the work is still in progress, and not necessarily ready to merge immediately. You can't merge a Draft PR directly—you have to move it out of draft first.
+        
+        When you see a draft PR (especially from outside the team!), treat it as though it's a working draft and reach out to the creator to discuss. 
+        
+        Read more on [GitHub's drafts](https://github.blog/2019-02-14-introducing-draft-pull-requests/). 
+      </td>
+    </tr>
+    </tbody>
+</table>
+
+As a **Hero or Sidekick**, make sure you attend to the following throughout your day:
+* Check in with the Hero/Sidekick at the start of your day (especially on Monday at the start of the week). Don’t forget to sync with the BCN hero if necessary.
+* Watch for incoming PRs in #docs_deploys, and review everything in the Needs triage column. Drag cards from that column to the appropriate column.
+* Work through the cards in the  Hero: to do column.
+
+**Everyone on the team** helps keep things moving:
+* All writers should keep an eye on the Any TW columns. 
+* There's one column for PRs that need a simpler review before merging (typo fixes, drive-by edits, etc), and another column for PRs that need a peer edit.
+* There are also two blocked columns: One for PRs blocked on a SME, and another column where we're waiting on the TW who created the PR to review feedback and/or merge.
```

Comment:
Would it make sense to move this in the board columns table `Waiting on SME`?

Reply:
I think this is more a checklist for writers, I'd keep it here.